src/controllers/playback/video/index.js

```diff
@@ -587,7 +587,12 @@ import { setBackdropTransparency, TRANSPARENCY_LEVEL } from '../../../components
 
         function showComingUpNextIfNeeded(player, currentItem, currentTimeTicks, runtimeTicks) {
             if (runtimeTicks && currentTimeTicks && !comingUpNextDisplayed && !currentVisibleMenu && currentItem.Type === 'Episode' && userSettings.enableNextVideoInfoOverlay()) {
-                const showAtSecondsLeft = runtimeTicks >= 3e10 ? 40 : runtimeTicks >= 24e9 ? 35 : 30;
+                let showAtSecondsLeft = 30;
+                if (runtimeTicks >= 3e10) {
```

Comment:
It would be nice to have comments to indicate what times these represent. :slightly_smiling_face:

Reply:
I added a few conversion constants which makes this a lot clearer to me. Let me know if you think that's a good approach.